pkg/internal/controllers/certificaterequestpolicies.go

```diff
@@ -256,30 +258,30 @@ func (c *certificaterequestpolicies) reconcileStatusPatch(ctx context.Context, r
 // Returns true if the condition has been updated or an existing condition has
 // been updated. Returns false otherwise.
 func (c *certificaterequestpolicies) setCertificateRequestPolicyCondition(
```

Comment:
@hrbasic I think you could use this function as an example here: https://github.com/cert-manager/approver-policy/blob/eafcc19aeb7e131047378a043b37a88a8661b61a/pkg/internal/controllers/certificaterequests.go#L253-L305